versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/persist/Reference.java

```diff
@@ -43,14 +44,23 @@ public interface Reference {
   @Value.Parameter(order = 4)
   boolean deleted();
 
+  @Value.Parameter(order = 5)
+  long createdAtMicros();
+
+  @Value.Parameter(order = 6)
+  @Nullable
+  @jakarta.annotation.Nullable
+  ObjId extendedInfoObj();
```

Comment:
Have you considered that future reference types could have more than one extended object id?

Reply:
No - why would you have that?

Reply:
I don't have concrete examples in mind. I was just wondering if it wouldn't be more future-proof to plan for that right now.